docs/intree-migrate.md

```diff
@@ -0,0 +1,229 @@
+persistentvolume/pvc-c4e7dca5-4be6-4168-8eb5-af6ade04261f   1Gi        RWO            Delete           Bound    default/testclaim1   fast-rbd                18s
+```
+
+### Mount Volume to a POD
+
+- Create a pod with PVC and verify the mount inside the POD
+
+```console
+$ kubectl create -f pod.yaml
+pod/task-pv-pod created
+
+$ kubectl get pods
+NAME                                         READY   STATUS    RESTARTS   AGE
+task-pv-pod                                  1/1     Running   0          2m36s
+
+$ mount |grep rbd
+/dev/rbd0 on /var/lib/kubelet/plugins/kubernetes.io/csi/pv/pvc-c4e7dca5-4be6-4168-8eb5-af6ade04261f/globalmount/0001-0020-b7f67366bb43f32e07d8a261a7840da9-0000000000000002-103b4240-7358-11ec-bcf8-0242ac110003 type ext4 (rw,relatime,stripe=16,_netdev)
+/dev/rbd0 on /var/lib/kubelet/pods/9a3b1f18-498e-4eae-8b6f-74736b3b90eb/volumes/kubernetes.io~csi/pvc-c4e7dca5-4be6-4168-8eb5-af6ade04261f/mount type ext4 (rw,relatime,stripe=16,_netdev)
+
+$ kubectl get pvc
+NAME         STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+testclaim1   Bound    pvc-c4e7dca5-4be6-4168-8eb5-af6ade04261f   1Gi        RWO            fast-rbd       4m40s
+
+$ kubectl exec -ti task-pv-pod -- df -kh |grep nginx
+/dev/rbd0                                976M  2.6M  958M   1% /usr/share/nginx/html
+```
+
+### Resize Volume
+
+- Resize PVC from 1Gi to 2Gi and verify the new size change in the POD
+
+```console
+$ kubectl edit pvc testclaim1
+persistentvolumeclaim/testclaim1 edited
+
+$ kubectl get pvc
+NAME         STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+testclaim1   Bound    pvc-c4e7dca5-4be6-4168-8eb5-af6ade04261f   2Gi        RWO            fast-rbd       6m41s
+
+$ kubectl get pv
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                STORAGECLASS   REASON   AGE
+pvc-c4e7dca5-4be6-4168-8eb5-af6ade04261f   2Gi        RWO            Delete           Bound    default/testclaim1   fast-rbd                6m38s
+
+$ kubectl exec -ti task-pv-pod -- df -kh |grep nginx
+/dev/rbd0                                2.0G  3.0M  2.0G   1% /usr/share/nginx/html
+
+```
+
+### Unmount Volume
+
+- Delete POD and verify unmount happened successfully
+
+```console
+$ kubectl get pods
+NAME                                         READY   STATUS    RESTARTS   AGE
+task-pv-pod                                  1/1     Running   0          5m31s
+
+$ kubectl delete pod task-pv-pod
+pod "task-pv-pod" deleted
+
+$ mount |grep rbd
+$
+
+```
+
+### Delete PVC
+
+- Delete PVC and verify PVC and PV objects are deleted
+
+```console
+$ kubectl delete pvc testclaim1
+persistentvolumeclaim "testclaim1" deleted
+
+$ kubectl get pvc
+No resources found in default namespace.
+
+$ kubectl get pv
+No resources found
+$
+
+```
+
+To know more about in-tree to CSI migration refer
```

Comment:
this is not part of "delete pvc", add an other heading for this reference. Maybe also link to the kubernetes docs about migration too.

Reply:
Done.